src/components/GestureButtons.tsx

```diff
@@ -1,65 +1,108 @@
-/* eslint-disable eslint-comments/no-unlimited-disable */
-/* eslint-disable */
-// @ts-nocheck TODO(TS) provide types
-import PropTypes from 'prop-types';
 import React from 'react';
-import { Animated, Platform, processColor, StyleSheet } from 'react-native';
+import {
+  Animated,
+  Platform,
+  processColor,
+  StyleSheet,
+  StyleProp,
+  ViewStyle,
+} from 'react-native';
 
 import createNativeWrapper from '../handlers/createNativeWrapper';
 import GestureHandlerButton from './GestureHandlerButton';
 import State from '../State';
 
-export const RawButton = createNativeWrapper(GestureHandlerButton, {
-  shouldCancelWhenOutside: false,
-  shouldActivateOnStart: false,
-});
+import {
+  GestureEvent,
+  HandlerStateChangeEvent,
+} from '../handlers/gestureHandlers';
+import {
+  NativeViewGestureHandlerPayload,
+  NativeViewGestureHandlerProperties,
+} from '../handlers/NativeViewGestureHandler';
+
+/* BUTTONS PROPERTIES */
+
+interface RawButtonProperties extends NativeViewGestureHandlerProperties {
+  exclusive?: boolean;
+  testID?: string;
```

Comment:
Why we add this here?

Reply:
It should be in `BaseButtonProperties`, as it is used in `BaseButton` in `GenericTouchable.tsx`. I will move it then. Is it what you meant?

Reply:
I mean, this is for tests and I'm not sure why we should support this prop. AFAIK this is quite common, if we should apply it to every exported component then maybe we should add some type that adds it?